### ISSUE TITLE
gg_point_col not working when numeric col_var is the same var as x or y

### ISSUE THREAD

**Issue by davidhodge931** [StatisticsNZ/simplevis#222]
```
library(simplevis)
library(palmerpenguins)

gg_point_col(penguins, bill_length_mm, body_mass_g, bill_length_mm)
```

**Comment by davidhodge931** [comment]
```
library(simplevis)
library(palmerpenguins)

# does not work
gg_point_col(penguins, bill_length_mm, body_mass_g, bill_length_mm)
gg_point_col(penguins, bill_length_mm, body_mass_g, body_mass_g)

# works
gg_point_col(penguins, bill_length_mm, body_mass_g, flipper_length_mm)
```

**Comment by davidhodge931** [comment]
Fixed 4.1.7 :)
